index.html

```diff
@@ -0,0 +1,304 @@
+              </ul>
+              <div class="action">
+                <button class="btn-default" type="button">See Project</button>
+              </div>
+            </div>
+          </article>
+          <article class="card bg-white">
+            <figure>
+              <img src="image/yoga.png" alt="yoga" />
+            </figure>
+            <div class="project-info">
+              <div class="project-header">
+                <h2 class="project-title">Tonic</h2>
+                <div class="header-details">
+                  <p class="client">CANOPY</p>
+                  <span class="separator-circle"></span>
+                  <p class="role">Back End Dev</p>
+                  <span class="separator-circle"></span>
+                  <p class="year">2015</p>
+                </div>
+              </div>
+              <p class="project-details">
+                A daily selection of privately personalized reads; no accounts
+                or sign-ups required.
+              </p>
+              <ul class="tags">
+                <li class="tag">html</li>
+                <li class="tag">css</li>
+                <li class="tag">javaScript</li>
+              </ul>
+              <div class="action">
+                <button class="btn-default" type="button">See Project</button>
+              </div>
+            </div>
+          </article>
+          <article class="card bg-white">
+            <figure>
+              <img src="image/art.png" alt="yoga" />
+            </figure>
+            <div class="project-info">
+              <div class="project-header">
+                <h2 class="project-title">Multi-Post Stories</h2>
+                <div class="header-details">
+                  <p class="client">CANOPY</p>
+                  <span class="separator-circle"></span>
+                  <p class="role">Back End Dev</p>
+                  <span class="separator-circle"></span>
+                  <p class="year">2015</p>
+                </div>
+              </div>
+              <p class="project-details">
+                A daily selection of privately personalized reads; no accounts
+                or sign-ups required.
+              </p>
+              <ul class="tags">
+                <li class="tag">html</li>
+                <li class="tag">css</li>
+                <li class="tag">javaScript</li>
+              </ul>
+              <div class="action">
+                <button class="btn-default" type="button">See Project</button>
+              </div>
+            </div>
+          </article>
+        </section>
+        <section class="about bg-white">
+          <h2 class="title">About Myself</h2>
+          <p class="details">
+            I’m a software developer! If you have a project you need coded, do not hestiate to
+            contact me.
+          </p>
+          <div class="social-media">
+            <a class="contact-link" href="#">LET’S CONNECT</a>
```

Comment:
- [ ] **[REQUIRED]** Please apply the requirement as suggested by the previous reviewer Kindly ensure  you use `hx` tag (h1,h2,h3,....)   Stick to the design as much as possible because as a developer it is very important peer attention to detail, your future employer will judge your work :+1:

Reply:
Done.